### PR TITLE
allow the winafl.dll path to be specified by -w in afl-tmin.exe

### LINKED ISSUE #179: PROGRAM ABORT : Unexpected result from pipe! expected 'P', instead received 'C'
afl-fuzz.exe` -D ..\..\..\..\DynamoRIO\bin32 -t 50000+ -i in -o out -- -target_module [ProgramPath].exe -target_offset 0x16fe4c -nargs 2 -- "[ProgramPath]" -E @@ C:\Users\[USER]\Downloads

[-] PROGRAM ABORT : Unexpected result from pipe! expected 'P', instead received 'C'

         Location : run_target(), c:\users\blackcow1987\downloads\fuzzing\winafl-master\afl-fuzz.c:2651

drrun.exe -version
drrun version 7.1.0 -- build 1

OS Version
Windows 10 Home(x64)

drrun.exe works normally, and I have confirmed that the coverage data is output.

### PR DESCRIPTION
**Pull request by cvspvr**
fixes #179 so that afl-tmin.exe can run from a different directory than the one winafl.dll is in

### CODE REVIEW

**Comment by ifratric** [comment]
Thanks for contributing! `winafl_dll_path` should be initialized to `winafl.dll`. For example, see this code from `afl-fuzz.c`: https://github.com/googleprojectzero/winafl/blob/master/afl-fuzz.c#L8469

**Comment by cvspvr** [comment]
whoops! i've added that in

File: afl-tmin.c [hunk 1]
```diff
@@ -1352,6 +1353,7 @@ int main(int argc, char** argv) {
   optind = 1;
   dynamorio_dir = NULL;
   client_params = NULL;
+  winafl_dll_path = NULL;
```
Comment (ifratric) [inline afl-tmin.c:1356]: It's going to be set to NULL here, overriding initialization. Either remove this line or add a NULL check later.

**Comment by cvspvr** [comment]
i'm really sorry! it should be good now

**Pull request merged**